Config.cpp

```diff
@@ -44,6 +44,13 @@ bool Config::readConfig(const std::string& fileName) {
     cmpIdx_ = (it != cfg.items().end())
       ? columnIdx[it->second.asString()] : -1;
 
+    it = cfg.find("loss_function");
+    if (it != cfg.items().end() && it->second.asString() == "logistic") {
+      lossFunction_ = L2Logistic;
```

Comment:
Do you wanna use a more clear name to identify GBM regression and Logistic Regression?